doc/flame/examples/lib/navigator.dart

```diff
@@ -0,0 +1,412 @@
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flutter/rendering.dart';
+
+class NavigatorGame extends FlameGame with HasTappableComponents {
+  late final Navigator navigator;
+
+  @override
+  Future<void> onLoad() async {
+    navigator = Navigator(
+      pages: {
+        'splash': Page(builder: SplashScreen.new),
+        'home': Page(builder: StartPageImpl.new),
+        'level1': LevelPage(builder: Level1PageImpl.new),
+        'level2': LevelPage(builder: Level2PageImpl.new),
+        'pause': Page(builder: PausePageImpl.new, transparent: true),
+      },
+      initialPage: 'splash',
+    )..addToParent(this);
```

Comment:
Maybe clearer to just wrap this in `add` instead, since the assigment can still be done at the same time?

Reply:
right, I keep forgetting about that syntax